### PR TITLE
Added first draft of the friend meetup events

### PR DESCRIPTION
**Pull request by jakubkrzysztofsikora**
#3

### CODE REVIEW

File: src/components/Subpage.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import Heading from 'grommet/components/Heading';
+import Grommet from 'grommet/components/Grommet';
+import Box from 'grommet/components/Box';
+
+import SubpageSection from './SubpageSection';
+import logo from "../images/logo.svg";
+
+export default ({title, children, className}) => <section className={`subpage ${className ? className : ''}`}>
```
Comment (BartlomiejGlac) [inline src/components/Subpage.jsx]: move tag to next line.

File: src/services/friend-events-provider.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        .map(friend => {
+            return ajax({
+                url: friend.eventProviderUrl,
+                method: 'GET',
+                crossDomain: false
+            }).pipe(map(data => {
+                switch (friend.eventProviderName) {
+                    case eventProviders.eventbrite:
+                        return data.response.events.map(event => {
+                            return {
```
Comment (BartlomiejGlac) [inline src/services/friend-events-provider.js]: remove return

File: src/services/friend-provider.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
+import eventProviders from '../constants/eventProviders';
+
```
Comment (BartlomiejGlac) [inline src/services/friend-provider.js]: empty line

Comment (jakubkrzysztofsikora) [reply src/services/friend-provider.js]: On purpose - to divide our imports and foreign imports

**Pull request merged**